Makefile

```diff
@@ -13,19 +13,28 @@ TESTARGS ?= -timeout 15m ./{cmd,pkg}/...
 
 CLI_ENV ?= "development"
 
-# TODO: This is duplicated in .goreleaser and we should figure out how to clean
```

Comment:
@Integralist is this TODO addressed or in the interest of de-duplicating the LDFLAGs should I completely remove LDFLAGS used for the dev builds (using `go build ..` directly)